CHANGELOG.md

```diff
@@ -9,9 +9,13 @@ and this project aims to adhere to [Semantic Versioning](http://semver.org/spec/
 ### Added
 ### Removed
 
+## [0.9.11] - 2019-09-23
```

Comment:
This should go in version.rb too, right?

Reply:
Yea I'm gonna update before merge.